lib/queue.js

```diff
@@ -305,6 +349,34 @@ export default class Queue {
     }).then(() => {
       return id;
     });
+  }
 
+  sendDelayedMessages() {
```

Comment:
Similar to the `WorkerPool` concept in here (although we stopped using it), it would be nice to expose some method like `checkForDelayedMessages(interval)` that just uses some cron library internally to check on the provided interval.